src/lib.rs

```diff
@@ -301,8 +300,8 @@ The `api` feature has been disabled, so the client API is not available. See ["C
 //!    With these macros, the two cases above would be solved like this:
 //!
 //!    - ```rust,ignore
-//!      // The compile_error!() is only emitted if 1.20 or lower is selected.
-//!      k8s_openapi::k8s_if_le_1_20! {
+//!      // The compile_error!() is only emitted if 1.21 or lower is selected.
+//!      k8s_openapi::k8s_if_le_1_21! {
 //!          compile_error!("This crate requires the v1_21 (or higher) feature to be enabled on the k8s-openapi crate.");
```

Comment:
The `compile_error` message doesn't match the macro being used any more. And in fact the whole example doesn't make sense if 1.21 is the lowest supported version. Need to find something new that's supported in 1.22+ and not in 1.21

Reply:
We should probably find something for 1.22 as 1.21 is EOL as well

Reply:
Yes, the later the better, since that gives us more time before we have to find a new one.

Reply:
I'll update the PR with 1.21 as well, and see if I can find something we can test upon